tests/test_autogen_environments.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+set -e
+
+for env in $(manage-commcare-cloud get environments)
+do
+    commcare-cloud ${env} aws-fill-inventory --cached &
+done
+
+wait
+
+git diff
+git update-index -q --refresh
+if git diff-index --quiet HEAD --; then
+    # No changes
+    exit 0
+else
+    # Changes
```

Comment:
Instead of `Changes` and `No changes` can you add a little more detail about what's changed? Might not be clear to someone who isn't familiar with `aws-fill-inventory` yet

Reply:
Offlined (pulling out "fail if there's a git diff" workflow came out of this)